src/field_player_avatar.c

```diff
@@ -78,17 +78,17 @@ static void PlayerAvatarTransition_ReturnToField(struct ObjectEvent * playerObje
 static bool8 PlayerIsAnimActive(void);
 static bool8 PlayerCheckIfAnimFinishedOrInactive(void);
 static bool8 player_is_anim_in_certain_ranges(void);
-static bool8 sub_805BF58(void);
+static bool8 player_is_anim_in_certain_ranges__running_state_not_turn(void);
```

Comment:
This is too long, vague, and not well formatted

Reply:
Any suggestions on what to call it then?

Reply:
I don't. If you don't have an actual name for it leave it as a `sub`